src/main/resources/application.yml

```diff
@@ -106,6 +106,34 @@ document:
             -   templateName: FL401-c8-welsh.docx
                 filename: FL401C8WelshDocument.pdf
                 document-generator: docmosis
+            -   templateName: PRL-ORDER-C21-COMMON.docx
+                filename: C21CreateOrder.pdf
+                document-generator: docmosis
+            -   templateName: PRL-ORDER-C21-DRAFT-COMMON.docx
+                filename: C21CreateOrderDraft.pdf
+                document-generator: docmosis
+            -   templateName: PRL-ORDER-C21-WELSH-COMMON.docx
+                filename: C21WelshCreateOrder.pdf
+                document-generator: docmosis
+            -   templateName: PRL-ORDER-C49-COMMON.docx
+                filename: C49CreateOrder.pdf
+                document-generator: docmosis
+            -   templateName: PRL-ORDER-C49-DRAFT-COMMON.docx
+                filename: C49CreateOrderDraft.pdf
+                document-generator: docmosis
+            -   templateName: PRL-ORDERS-N117-DRAFT.docx
+                filename: N117CreateOrderDraft.pdf
+                document-generator: docmosis
+            -   templateName: PRL-ORDERS-N117-FINAL.docx
+                filename: N117CreateOrder.pdf
+                document-generator: docmosis
+            -   templateName: PRL-ORDER-FL404b-DRAFT-COMMON.docx
+                filename: FL404bCreateOrderDraft.pdf
```

Comment:
Please rename the file with name matching to order label in docmosis template. e.g "amended_discharged_or_varied_fl404b_order_draft.pdf"